vue-frontend/src/components/flutter-app-development/LandingSection.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="tw-container md:tw-text-left tw-text-center md:tw-flex xl:tw-my-8 tw-pb-16"
+  >
+    <div
+      class="tw-flex tw-flex-col tw-gap-4 md:tw-gap-6 md:tw-flex-1 md:tw-w-[34rem]"
+    >
+      <h1 class="header-1">Flutter App Development Company</h1>
+      <p class="sub-h1-regular tw-text-black-900 tw-text-opacity-60">
+        We harness the power of Flutter to create visually stunning, fast, and
+        highly responsive applications that effortlessly work on both iOS and
+        Android platforms. Our team turns your unique business requirements into
+        reality, giving your brand the competitive edge it needs in this dynamic
+        mobile app market.
+      </p>
+      <div class="tw-mt-4 tw-hidden md:tw-block">
+        <button class="gradient-btn">
+          <router-link
+            to="/contact"
+            @click.native="mixpanel.track('tap_flutter_development_cta')"
+          >
+            <span class="sub-h4-semibold tw-border-none"
+              >Get Free Consultation</span
+            >
+          </router-link>
+        </button>
+      </div>
+    </div>
+    <div
+      class="tw-mt-[2.01rem] tw-flex md:tw-flex-wrap tw-flex-col tw-justify-center tw-gap-[0.8rem] md:tw-flex-1 md:tw-relative tw-pb-16"
+    >
+      <img
+        :src="landing400"
+        :srcset="`${landing400} 400w,${landing800} 800w`"
+        class="tw-mx-auto md:tw-hidden tw-w-[14.5625rem] tw-h-[13.57931rem]"
+        alt="flutter-landing"
+      />
+      <img
```

Comment:
don't use these 2 image tags as if you give all width here it will take image according to device width